### ISSUE TITLE
{{No talkback}} category

### ISSUE THREAD

**Issue by JJMC89** [azatoth/twinkle#405]
Per a [CfD for Category:Users who do not wish to receive talkbacks](https://en.wikipedia.org/wiki/Wikipedia:Categories_for_discussion/Log/2017_October_12#Category:Users_who_do_not_wish_to_receive_talkbacks), Twinkle should be modified to not depend on the category. If this is not possible without loss of functionality, the category will be renamed to `Wikipedians who wish to not receive talkbacks`.

The current category name is referenced:
https://github.com/azatoth/twinkle/blob/67e51a0754d29c78ef189d8731d8f2633285e3bd/modules/friendlytalkback.js#L89

**Comment by JJMC89** [comment]
@MusikAnimal @atlight Is it possible to remove the category check without loss of functionality?

**Comment by atlight** [comment]
Yes, it's possible; we'd have to run the API query unconditionally. Let me see what I can do.

**Comment by atlight** [comment]
Hopefully fixed; someone like @MusikAnimal will need to synchronise this.
